scripts/wdl/README.md

```diff
@@ -59,6 +60,8 @@ Reference used must be the same between PoN and case samples.
 - ``case_gatk_acnv_workflow.enable_gc_correction`` -- "true" is recommended. This is a true/false option change to false to bypass GC bias correction step.  *This must be the same as was used to generate the PoN*
 - ``case_gatk_acnv_workflow.isWGS`` -- Change to true if running WGS samples.  You should also change ``case_gatk_acnv_workflow.seg_param_undoSplits`` and ``case_gatk_acnv_workflow.seg_param_undoSD`` to reduce (not eliminate) hyper-segmentation.
 - ``case_gatk_acnv_workflow.wgsBinSize`` -- "10000" is recommended.  Size of bins (in bp) for WGS read counts CLI.  *This must be the same as was used to generate the PoN*.  Ignored if not running WGS.
+- ``case_gatk_acnv_workflow.stringency`` -- (Advanced) ``GetHetBayesianCoverage`` parameter, default=30
```

Comment:
You should add a note that end users do not ever really have to change these values.